dotnetAvalonia/.conf/deps.json

```diff
@@ -3,6 +3,7 @@
         "openssh-client",
         "sshpass",
         "dotnet-sdk-6.0",
+        "dotnet-sdk-7.0",
```

Comment:
Should it be dependet on both "dotnet-sdk-7.0" and "dotnet-sdk-6.0"?

Reply:
yeah, because the application will be compiled using the dotnet 6, but the extension use dotnet 7